### ISSUE TITLE
Profile Icons display incorrectly after restarting the browser

### ISSUE THREAD

**Issue by Brave-Matt** [brave/brave-browser#13552]
## Description 

Brave shows strange avatars/icons when switching profiles after launching the browser. Note that the _second_ time the profile menu is opened, the profile icons will display without issue:
![image](https://user-images.githubusercontent.com/3299098/104490540-d192c400-5585-11eb-8b2d-217630677886.png)
![image](https://user-images.githubusercontent.com/3299098/104490578-da839580-5585-11eb-8028-d0f9b986b098.png)

## Steps to Reproduce

   1. Launch/restart browser
   2. Open browser profile menu
   3. Observe icons
   4. Close/reopen the menu -- observe icons now displaying correctly.

## Expected result:
Icons should always display correctly.

## Reproduces how often: 

Unclear.

## Brave version (brave://version info)

Windows 10
Brave v1.18.77

## Version/Channel Information:

- Can you reproduce this issue with the current release? N/A
- Can you reproduce this issue with the beta channel? N/A
- Can you reproduce this issue with the nightly channel? N/A

## Miscellaneous Information:

Original user thread:
https://community.brave.com/t/profile-icons-avatars-change-when-brave-restarts/189470/6

**Comment by JohnEricson** [comment]
I got the same problem today when I updated Brave on two computers. I can no longer tell the difference between my Brave windows profiles in the Windows 10 taskbar, which really impacts my productivity, as I have to search all my different windows for the one I'm looking for.

![image](https://user-images.githubusercontent.com/946327/105400806-b5e28a00-5c25-11eb-882d-a7ae26bd0162.png)
This is 3 different profiles in Brave in the Windows 10 taskbar and I can no longer tell the difference between them.

OS: Windows 10 Pro (build: 19042.746)
Brave: Version 1.19.86 Chromium: 88.0.4324.96 (Official Build) (64-bit)

**Comment by wsor4035** [comment]
> I got the same problem today when I updated Brave on two computers. I can no longer tell the difference between my Brave windows profiles in the Windows 10 taskbar, which really impacts my productivity, as I have to search all my different windows for the one I'm looking for.
> 
> ![image](https://user-images.githubusercontent.com/946327/105400806-b5e28a00-5c25-11eb-882d-a7ae26bd0162.png)
> This is 3 different profiles in Brave in the Windows 10 taskbar and I can no longer tell the difference between them.
> 
> OS: Windows 10 Pro (build: 19042.746)
> Brave: Version 1.19.86 Chromium: 88.0.4324.96 (Official Build) (64-bit)

I am also having the same issue

**Comment by w-** [comment]
does this have something to do with how the old profile icons/avatars are no longer selectable?

**Comment by JohnEricson** [comment]
> does this have something to do with how the old profile icons/avatars are no longer selectable?

That is absolutely a possibility. In my case I used the old avatars when I got the problem. As a workaround I could tell my profiles apart again when I went into the profile of each and changed the avatar to one of the newly available ones. But still in the file systems I could find the images for the old avatars, but somehow Brave isn't using them.

**Comment by wsor4035** [comment]
at least for me attempting to even change the profile image causes brave to crash

**Comment by simonhong** [comment]
@wsor4035 Did you see same crash with https://github.com/brave/brave-browser/issues/13762?

**Comment by wsor4035** [comment]
yup, i was attempting to change it to see if that would fix the issue as JohnEricson alluded to

**Comment by simonhong** [comment]
@wsor4035 Thanks for checking 👍
That crash fix was already merged into nightly(`1.22.x`) and will be uplifted to beta(`1.21.x`).

**Comment by wsor4035** [comment]
thanks, ill wait till its available 👍

**Comment by wsor4035** [comment]
just updated to 1.21.73 which allowed me to change the image without crashing which fixed the taskbar icons, thanks 👍
